charts/osbuilder/Chart.yaml

```diff
@@ -8,5 +8,5 @@ maintainers:
 - name: Ettore Di Giacinto
   email: mudler@kairos.io
 
-version: 0.5.0
-appVersion: "v0.5.0"
+version: 0.5.1
+appVersion: "v0.5.1"
```

Comment:
This is a "look-ahead" assuming this is the next version of the osbuilder image.

Reply:
`quay.io/kairos/osbuilder-tools:v0.5.1` is there now (since this: https://github.com/kairos-io/osbuilder/actions/runs/3730265686/jobs/6327114779)